### ISSUE TITLE
zom.im and riot desktop issue's. 

### ISSUE THREAD

**Issue by githubboy1** [vector-im/riot-web#10588]
Hello

So i heard that zom.im is using the matrix protocol now. 
So i have tried riot desktop messenger (matrix.org) connecting with zom android messenger (:nang.zom.im) 
Well thats works, and you can connecting/talking to each other. 
But there are few problems and i am not sure this is a Zom or Riot/matrix problem.
But let me tell you anyways. 

First there is a verify problem. 
i can choose a two way verify verification (doesnt work) 
And can usa a legacy verify verification. 
This last one, is a problem because you must do it again and again when you log in/out with riot. So thats must me saved somehow.. 

Second problem, when you log out with riot and log in again, i cant read chat history. 
it said: ** Unable to decrypt: The sender's device has not sent us the keys for this message. **
When i click 'Re-request encryption keys' nothing happens, not in riot, not in zom. 

question, after setting Encryption "on"  in security & privacy tab, are this messages still encrypted and secured between zom and riot?

**Comment by turt2live** [comment]
It looks like zom.im doesn't support the new verification method at all - contacting their support for it would be better.

The logging out problem is your device losing keys for encrypted messages. Verifying devices does not help keep keys around: avoid logging out, or use key backup to maintain your keys. 

Once encryption is enabled, it cannot be disabled. If they have implemented encryption correctly, they are end-to-end encrypted forever.

**Comment by githubboy1** [comment]
i notice that Riot Dekstop app for linux have the same issue. 
When log out,  and someone send a message to me i cant read this message when i log in again. 
it says:  ** Unable to decrypt: The sender's device has not sent us the keys for this message.**
and must verify again.

Why is that? on android i don't have this issue. 
Also not when i turn my phone completely off. 
When i turn it on again, i can read all my missed messages.

**Comment by t3chguy** [comment]
When you log in you generate new keys, turning your phone off and on doesn't require you to log into riot again. Don't log out of riot if you want your cryptographic identity to stay alive.

**Comment by githubboy1** [comment]
"turning your phone off and on doesn't require you to log into riot again"
Correct!
But why is cryptographic identity stay alive on the phone when you log in and out and not on desktop version?

**Comment by MayeulC** [comment]
You do not have to log out of the desktop version. Just close the application.

Logging out is designed to remove the keys from the computer, so you would have to generate new ones after you reconnect.
If you choose to log out on your phone (that option exists as well), you will lose the keys as well.

Turn of your computer without logging out. You should be able to find your messages after turning it on again.

I guess you are looking for #10224 

So, to summarize once again: if you want to delete your keys from the computer, that's what the logout option is for. If you want to keep your keys, but close the application -- just close the application.

**Comment by jryans** [comment]
If I am understanding correctly, I don't believe there's anything for the Riot team to do in this issue. It appears to be a mix of support questions and issues with zom.im.
